api/openapi.gen.json

```diff
@@ -28,8 +28,10 @@
           "instances": [
             {
               "detail": {
-                "publicdns": "",
-                "publicipv4": "10.0.0.88"
+                "privateipv4": "172.31.36.10",
+                "privateipv6": "",
```

Comment:
can we add this to example as well, or does this default to blank?

Reply:
You mean IPv6, I can add one, sure. It is unconfigured on new accounts of course :-D